stories/Appointment.stories.tsx

```diff
@@ -62,6 +64,7 @@ const Template: Story<StoryAppointmentProps> = ({ ...args }) => {
 }
 
 export const Appointment = Template.bind({})
+const exampleDate = DateTime.now().toString()
 Appointment.args = {
-  date: new Date().toJSON(),
+  date: exampleDate,
```

Comment:
TypeScript got mad if I did this inline due to it being confused about the `:` syntax

Reply:
```typescript
  date: exampleDate,
```

The typescript parser, at least on my machine, got a little confused when I made it

```typescript
  date: DateTime.now().toString()
```

complaining the that type declaration `DateTime` didn't have a `now()` method - it probably wouldn't be a big problem but I just avoided it

Reply:
Oh yeah. I see. I've run into this before, too. 

I was confused by what you meant by the `:` syntax, but I see that you're referring to the property syntax.